homeassistant/components/zha/sensor.py

```diff
@@ -341,9 +341,11 @@ def extra_state_attributes(self) -> dict[str, Any]:
     def formatter(self, value: int) -> int | float:
         """Return 'normalized' value."""
         multiplier = getattr(
-            self._cluster_handler, f"{self._div_mul_prefix}_multiplier"
+            self._cluster_handler, f"{self._div_mul_prefix}_multiplier", self._multiplier
+        )
+        divisor = getattr(
+            self._cluster_handler, f"{self._div_mul_prefix}_divisor", self._divisor
         )
```

Comment:
This is so that if an ElectricalMeasurement sensor class sets _div_mul_prefix = None the formatter will either use the sensor's specified _multiplier/divisor values or the default ElectricalMeasurement values of 1.

Reply:
This is hacky. It will try to get an attribute called "None_multiplier". It should check it for none ness first

Reply:
Good point, addressed in https://github.com/home-assistant/core/pull/107641/commits/2eab8b4e79c8c7b0bf52756962aedde09a84df97